admin/pre_reg/views.py

```diff
@@ -209,3 +222,21 @@ def view_file(request, node_id, provider, file_id):
     fp = FileNode.load(file_id)
     wb_url = fp.generate_waterbutler_url()
     return redirect(wb_url)
+
+
+def get_metadata_files(draft):
+    for q in ['q7', 'q11', 'q12', 'q13', 'q16', 'q19', 'q26']:
```

Comment:
Can we avoid hardcoding these? Can you inspect the metadata (i.e. questions that have `/value/uploader/extra`) instead?